src/pages/Productdetail/Productdetail.js

```diff
@@ -1,115 +1,114 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import ImageSlider from './ImageSlider';
+import ProductDetailInfo from './productDetailInfo';
 import './Productdetail.scss';
 
 class Productdetail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      detailData: {},
+    };
+  }
+
+  componentDidMount() {
+    console.log('hello');
+    console.log(this.state.detailData);
+    fetch('data/ItemData.json')
```

Comment:
```suggestion
    fetch('/data/ItemData.json')
```
앞에 / 붙여서 절대경로 설정해주셔야 합니다!
안붙이면 현재 url기준으로 계산하기 때문에 문제가 발생할 수 있습니다.

Reply:
네 절대경로로 수정했습니다~!